modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase

```diff
@@ -136,74 +98,54 @@ definition {
 		}
 	}
 
-	@description = "LPS-164733 - Verify that the EVP Staff can approve a new organization"
+	@description = "LPS-164733 - Verify that the EVP Manager can approve a new organization and see the comment in details page"
 	@priority = 4
-	test ManagerCanApproveOrganization {
-		task ("Given that Sign as Manager user") {
-			User.logoutAndLoginPG(
-				userLoginEmailAddress = "manager@liferay.com",
-				userLoginFullName = "Manager User");
+	test ManagerCanApproveOrganizationWithComment {
+		task ("When Manager user approves the created organization") {
+			EVPUtils.managerApproval(
+				listingPage = "Organization",
+				organizationName = "Organization Test");
 		}
 
-		task ("And go to organization page") {
-			EVPUtils.gotoEVP();
-
-			EVPNavigator.goToNavItem(navItem = "Organization");
+		task ("Then the status will be updated to 'Awaiting Finance Approval'") {
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Awaiting Finance Approval");
 		}
 
-		task ("When the manager approve the organization created") {
+		task ("When Manager user goes to details page") {
 			EVPOrganization.kebabMenuOrganization(organizationName = "Organization Test");
 
-			MenuItem.click(menuItem = "Review");
-
-			EVPUtils.changeStatus(
-				button = "Approve",
-				typeMessage = "Your organization has been approved!");
+			MenuItem.click(menuItem = "Details");
 		}
 
-		task ("Then the status will be updated to 'Awaiting Finance Approval'") {
-			EVPUtils.viewStatus(
-				organizationName = "Organization Test",
-				status = "Awaiting Finance Approval");
+		task ("Then the manager user can see the comment 'Your organization has been approved!' in the details page") {
+			EVPUtils.assertHeadersTitle(listHeaderTitle = "Organization Test,Your organization has been approved!");
 		}
 	}
 
 	@description = "LPS-164733 - Verify that the Manager can not see 'Review button' when organization are in 'Awaiting Finance Approval' status"
 	@priority = 4
 	test ManagerCannotSeeReviewButtonInAwaitingFinanceStatus {
-		task ("Given that Sign as Manager user") {
-			User.logoutAndLoginPG(
-				userLoginEmailAddress = "manager@liferay.com",
-				userLoginFullName = "Manager User");
-		}
-
-		task ("And go to organization page") {
-			EVPUtils.gotoEVP();
-
-			EVPNavigator.goToNavItem(navItem = "Organization");
+		task ("Given that the created Organization has already been approved by Manager user") {
+			EVPUtils.managerApproval(
+				listingPage = "Organization",
+				organizationName = "Organization Test");
 		}
 
-		task ("When the manager approve the organization created") {
+		task ("When the manager user clicks on kebab menu of a Organization with 'Awaiting Finance Approval' status") {
```

Comment:
Fix typo "When the manager user clicks on kebab menu of an Organization with 'Awaiting Finance Approval' status"